pkg/nugetconfig/parse.go

```diff
@@ -0,0 +1,54 @@
+package nugetconfig
+
+import (
+	"encoding/xml"
+	"github.com/aquasecurity/go-dep-parser/pkg/types"
+	"golang.org/x/xerrors"
+	"io"
+)
+
+type Dependencies map[string]Dependency
+
+type Dependency struct {
+	Type     string
+	Resolved string
+}
+
+type cfgPackageReference struct {
+	XMLName         xml.Name `xml:"package"`
+	TargetFramework string   `xml:"targetFramework,attr"`
+	Version         string   `xml:"version,attr"`
+	ID              string   `xml:"id,attr"`
```

Comment:
We should add `developmentDependency` and skip it when it is true.
https://docs.microsoft.com/en-us/nuget/reference/packages-config